src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -456,6 +458,8 @@ std::string ContextAttribute::render
       std::string effectiveValue      = "";
       bool        valueIsNumberOrBool = false;
 
+      using namespace orion;
+
```

Comment:
using namespace orion *twice*.

Perhaps this 'directive' should be in the beginning of the file, right after includes.
That way the risk to repeat it becomes very small.